examples/ASP.NET Core 6/ASP.NET Core 6 NLog Example/nlog.config

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<nlog xmlns="http://www.nlog-project.org/schemas/NLog.xsd"
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      autoReload="true"
+      internalLogLevel="Info"
+      throwConfigExceptions="true"
+      internalLogFile="c:\temp\internal-nlog-AspNetCore6.txt">
+
+  <!-- enable asp.net core layout renderers -->
+  <extensions>
+    <add assembly="NLog.Web.AspNetCore"/>
+  </extensions>
+
+  <!-- the targets to write to -->
+  <targets>
+    <!-- File Target for all log messages with basic details -->
+    <target xsi:type="File" name="allfile" fileName="c:\temp\nlog-AspNetCore6-all-${shortdate}.log"
+            layout="${longdate}|${event-properties:item=EventId_Id:whenEmpty=0}|${uppercase:${level}}|${logger}|${message} ${exception:format=tostring}" />
+
+    <!-- File Target for own log messages with extra web details using some ASP.NET core renderers -->
+    <target xsi:type="File" name="ownFile-web" fileName="c:\temp\nlog-AspNetCore6-own-${shortdate}.log"
+            layout="${longdate}|${event-properties:item=EventId_Id:whenEmpty=0}|${uppercase:${level}}|${logger}|${message} ${exception:format=tostring}|url: ${aspnet-request-url}|action: ${aspnet-mvc-action}|${callsite}| body: ${aspnet-request-posted-body}" />
+
+    <!--Console Target for hosting lifetime messages to improve Docker / Visual Studio startup detection -->
+    <target xsi:type="Console" name="lifetimeConsole" layout="${level:truncate=4:lowercase=true}\: ${logger}[0]${newline}      ${message}${exception:format=tostring}" />
```

Comment:
Could this be updated to match the wiki-page, so it becomes:
```xml
<target xsi:type="Console" name="lifetimeConsole" layout="${MicrosoftConsoleLayout}" />
```

Reply:
Sure thing. The `nlog.config` file in the ASP.NET Core 6 sample is a copy from the ASP.NET Core 5 sample with only the paths updated. I can upgrade it in the sample if you prefer the layout from the Wiki. We should probably do that for the ASP.NET Core 5 sample too, right?

Reply:
Yes and that change is already pending with #734